07-routing-gossip.md

```diff
@@ -565,6 +567,11 @@ the channel when the peer reestablishes contact.  Because gossip
 messages are batched and replace previous ones, the result may be a
 single seemingly-redundant update.
 
+When a node creates a new `channel_update` to change its channel parameters,
+it will take some time to propagate through the network and payers may use
+older parameters. It is recommended to keep accepting older parameters for a
+while to improve payment latency and reliability.
```

Comment:
Maybe we should define a number here? It seems 10 minutes are reasonable?

Reply:
Done.